<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicionIncertidumbre</class>
 <widget class="QDialog" name="MedicionIncertidumbre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MEDICIÓN E INCERTIDUMBRE</string>
  </property>
  <widget class="QLabel" name="lb_regla">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mostrar_objeto">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>601</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>340</y>
     <width>84</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_md">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Menor división</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_md_1cm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1 cm</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_md_01cm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0.1 cm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_medir">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(159, 159, 159);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>MEDIR</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>379</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Objetos</string>
   </property>
   <widget class="QPushButton" name="btn_i_obj1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_i_obj2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_i_obj3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="btn_volver">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(255, 0, 0);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string> VOLVER</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_volver.png</normaloff>:/new/prefix1/recursos/icono_volver.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;&amp;quot;Coloque la regla correctamente para medir&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
